feature-staking-impl/src/main/java/jp/co/soramitsu/feature_staking_impl/data/network/subscan/StakingApi.kt

```diff
@@ -22,4 +25,14 @@ interface StakingApi {
         @Path("subDomain") subDomain: String,
         @Body body: StakingRewardRequest,
     ): SubscanResponse<StakingRewardHistory>
+
+    @POST("//api.subquery.network/sq/OnFinality-io/sum-reward-kusama")
+    suspend fun getSumRewardKusama(
+        @Body body: StakingSumRewardRequest
+    ): SubQueryResponse<SumRewardResponse>
+
+    @POST("//api.subquery.network/sq/OnFinality-io/sum-reward")
+    suspend fun getSumRewardDot(
+        @Body body: StakingSumRewardRequest
+    ): SubQueryResponse<SumRewardResponse>
```

Comment:
Why not `//api.subquery.network/sq/OnFinality-io/{path}` with `path` variable? This way there will be no need in separate method for kusama